shell/platform/android/io/flutter/FlutterInjector.java

```diff
@@ -88,6 +93,11 @@ public DeferredComponentManager deferredComponentManager() {
     return deferredComponentManager;
   }
 
+  @NonNull
+  public FlutterJNI.Factory flutterJNIFactory() {
```

Comment:
nit: isn't it idiomatic in java to name this `getFlutterJNIFactory()`?

Reply:
you're right